### DISCUSSION TITLE
Following

### DISCUSSION THREAD

**Discussion by L10N37** [johnbaumann/OpenMC#14] [General]
I've spent most of my spare time this week trying to get into this same project. Got a board here with WROOM32, SD card on VSPI and PSX wired to HSPI and another SD card on the MMC.. nothing but issues with both my ESP32's.. both bricked. One in a  boot loop. Just googled to see if ANYONE else had done anything like this that was open source and found this! I'm an absolute beginner but looking forward to getting some new boards and seeing what stage this is up to. I will be doing mine from scratch, but I'll keep my eye on this and reference it if i use any ideas off it. 

My first issue arose instantly, I didn't want to use the flash as an in between from the console to the SD card, but even though these are stated to have a certain amount of RAM (over 500Kb) you can't fit an entire 15 blocks (128Kb) in an SRAM buffer/ structure.

Anyway, I'm an absolute beginner. Couple projects on a normal Arduino then decided to jump into this and having a memcarduino that I often use, figured there was enough info out there (especially with no cash having all the command bytes and returns etc listed)

I figured step one was to just stream memory card reads and writes to the serial monitor and get a grasp of what's happening, then after that get the ESP32 to ignore anything until it received the memory card flag and sort of puzzle it together slowly from there with the help of memcarduino source / no$ information. Got a shock when I found this....

Cheers

:)

**Comment by johnbaumann** [comment]
Hi, glad you've found the project! I've been working on different versions of this for nearly a year, initially this actually started on an Arduino(https://github.com/johnbaumann/VirtualMC). Memcarduino was a big help in getting started, between reading the source and using it as a device to communicate with. The no$ docs were also infinitely helpful. There's actually an easy to solder template PCB in the works right now. Feel free to join the psxdev discord(I post a lot more pics+info there), there's some really great people there that have helped me out with this project along the way!
https://discord.gg/QByKPpH
